src/tezos/TezosEngine.ts

```diff
@@ -387,13 +386,13 @@ export class TezosEngine extends CurrencyEngine<
     // eslint-disable-next-line @typescript-eslint/no-floating-promises
     this.addToLoop('checkTransactionsInnerLoop', TRANSACTION_POLL_MILLISECONDS)
     // eslint-disable-next-line @typescript-eslint/no-floating-promises
-    super.startEngine(opts)
+    super.startEngine()
```

Comment:
This could be `await super.startEngine()`, and then we can delete the comment above.